Formula/wine.rb

```diff
@@ -64,53 +51,421 @@ def require_universal_deps?
+                              "--prefix=#{libexec}",
+                              "--disable-static",
+                              *depflags
+        system "make", "install"
+      end
+
+      resource("webp").stage do
+        system "./configure", "--disable-dependency-tracking",
+                              "--prefix=#{libexec}",
+                              "--disable-static",
+                              "--disable-gl",
+                              "--enable-libwebpmux",
+                              "--enable-libwebpdemux",
+                              "--enable-libwebpdecoder",
+                              *depflags
+        system "make", "install"
+      end
+
+      resource("fontconfig").stage do
+        system "./configure", "--disable-dependency-tracking",
+                              "--prefix=#{libexec}",
+                              "--disable-static",
+                              "--with-add-fonts=/System/Library/Fonts,/Library/Fonts,~/Library/Fonts",
+                              "--localstatedir=#{var}",
+                              "--sysconfdir=#{prefix}",
+                              *depflags
+        system "make", "install", "RUN_FC_CACHE_TEST=false"
+      end
+
+      resource("gd").stage do
+        # Poor man's patch as this is a resource
+        inreplace "src/gd_gd2.c",
+                  "#include <math.h>",
+                  "#include <math.h>\n#include <limits.h>"
+        system "./configure", "--disable-dependency-tracking",
+                              "--prefix=#{libexec}",
+                              "--disable-static",
+                              "--without-x",
+                              "--without-xpm",
+                              "--with-png=#{libexec}",
+                              "--with-fontconfig=#{libexec}",
+                              "--with-freetype=#{libexec}",
+                              "--with-jpeg=#{libexec}",
+                              "--with-tiff=#{libexec}",
+                              "--with-webp=#{libexec}"
+        system "make", "install"
+      end
+
+      resource("libgphoto2").stage do
+        system "./configure", "--disable-dependency-tracking",
+                              "--prefix=#{libexec}",
+                              *depflags
+        system "make", "install"
+      end
+
+      resource("net-snmp").stage do
+        # https://sourceforge.net/p/net-snmp/bugs/2504/
+        ln_s "darwin13.h", "include/net-snmp/system/darwin14.h"
+        ln_s "darwin13.h", "include/net-snmp/system/darwin15.h"
+        ln_s "darwin13.h", "include/net-snmp/system/darwin16.h"
+
+        system "./configure", "--disable-debugging",
+                              "--prefix=#{libexec}",
+                              "--disable-static",
+                              "--enable-ipv6",
+                              "--with-defaults",
+                              "--with-persistent-directory=#{var}/db/net-snmp",
```

Comment:
Same sort of comment here as above. Not sure it's sensible to have things share these permanent directories when they're unlikely to be kept perfectly in-sync over time.